lib/plenario_web/controllers/web/page_controller.ex

```diff
@@ -28,15 +28,12 @@ defmodule PlenarioWeb.Web.PageController do
   end
 
   defp parse_date(conn, date_string) do
-    case DateTime.from_iso8601("#{date_string}T00:00:00.0Z") do
-      {_, datetime, _} ->
-        {datetime, conn}
-      {:error, :invalid_format} ->
-        {nil, put_flash(conn, :error,
-          "Invalid date #{date_string}, must be formatted as YYYY-MM-DD")}
-      {:error, :invalid_date} ->
+    case NaiveDateTime.from_iso8601("#{date_string}T00:00:00.0") do
+      {:error, _} ->
         {nil, put_flash(conn, :error,
-          "Invalid date #{date_string}, date doesn't exist")}
+        "Invalid date #{inspect(date_string)}")}
+      {_, datetime} ->
+        {datetime, conn}
```

Comment:
But why :slightly_frowning_face: - I understand the want for terseness but it's a few lines of code at the expense of more informative messages.

Reply:
Me being fussy. It can be added back if you really miss it that much :)